src/ripple/nodestore/backend/CassandraFactory.cpp

```diff
@@ -241,16 +239,14 @@ class CassandraBackend : public Backend
             Throw<std::runtime_error>(ss.str());
         }
 
-        std::string username = get<std::string>(config_, "username");
+        std::string username = get(config_, "username");
         if (username.size())
         {
-            std::cout << "user = " << username.c_str() << " password = "
-                      << get<std::string>(config_, "password").c_str()
+            std::cout << "user = " << username.c_str()
```

Comment:
Is there a benefit to invoking `c_str()`? Also applies to lines 246 and 249.

Reply:
Good eyes, @undertome.  The `.c_str()` calls on lines 245 and 246 serve no useful purpose.

The `.c_str()` calls on line 249 I left alone, however.  I believe the database folks are invoking a function that expects a pointer to a null terminated C-style string.  Things _compile_ if I remove the calls to `c_str()`.  But I'm not 100% confident that still provides the desired behavior.  So I'm inclined to leave it alone.

Reply:
`cass_cluster_set_credentials()` expects the params to be `const char *`. The other two uses are fine to change. This whole file is only compiled if the `RIPPLED_REPORTING` macro is defined.